dockerfile_parse/parser.py

```diff
@@ -262,9 +262,11 @@ def _clean_comment_line(line):
 
         instructions = []
         lineno = -1
+        line_continuation_char = '\\'
         insnre = re.compile(r'^\s*(\S+)\s+(.*)$')  # matched group is insn
         contre = re.compile(r'^.*\\\s*$')          # line continues?
         commentre = re.compile(r'^\s*#')           # line is a comment?
+        directivere = re.compile(r'^\s*#\s*escape\s*=\s*(\\|`)\s*$', re.I)  # escape directive
```

Comment:
Dockerfile supports more (2 for now) directives, can we name it `escape_directive_re`?